models/labels/labels_all.sql

```diff
@@ -0,0 +1,8 @@
+{{ config(
+    alias = 'all',
+    materialized = 'table',
+    file_format = 'delta')
+}}
+
+-- Static Labels
+SELECT * FROM {{ ref('labels_cex') }}
```

Comment:
what are the future use cases of adding labels here, will they have the same columns and be able to just ```select *``` union?

Reply:
i think #1513 may answer this as well, looks like you'll always conform to that standard

Reply:
yes, that's also right 👍